create_diracosrc.sh

```diff
@@ -16,9 +16,42 @@
     echo 'export DAVIX_USE_LIBCURL=1'
     echo ''
     echo '# Set up the X509 variables'
-    echo "export X509_CERT_DIR=\${X509_CERT_DIR-'$PREFIX/etc/grid-security/certificates'}"
-    echo "export X509_VOMS_DIR=\${X509_VOMS_DIR-'$PREFIX/etc/grid-security/vomsdir'}"
-    echo "export X509_VOMSES=\${X509_VOMSES-'$PREFIX/etc/grid-security/vomses'}"
+    echo ''
+    echo '# Function check if folder exist and contins files'
```

Comment:
```suggestion
    echo '# Function check if folder exist and contains files'
```

Reply:
Thanks, fixed. That probably means the same typo is all over the `dirac-install` in DIRAC :D